### PR TITLE
Make ad title and thumbnail go to url if they have no selftext

### PR DESCRIPTION
**Pull request by curioussavage**
👓 @nramadas @madbook 

High priority por favor this is hurting click through numbers.

### CODE REVIEW

File: src/app/components/Post/PostHeader/index.jsx [hunk 1]
```diff
@@ -327,6 +327,15 @@ function renderPostTitleLink(post, showLinksInNewTab) {
   const titleLinkClass = `PostHeader__post-title-line ${post.visited ? 'm-visited' : ''}`;
   const target = linkExternally && showLinksInNewTab ? '_blank' : null;
 
+  if (post.promoted && !post.isSelf) {
+    //promoted posts have their own tracking
+    return (
+      <a className={ titleLinkClass } href={ post.cleanUrl } target={ target }>
```
Comment (nramadas) [inline src/app/components/Post/PostHeader/index.jsx]: Aren't these still urls that mobile web can handle? We can re-use the `<Anchor>` template that get's normally returned below, and just add the `post.promoted && !post.isSelf` logic to line 324.

**Comment by nramadas** [comment]
:haircut_woman:

**Comment by nramadas** [comment]
:nail_care:

File: src/app/components/Post/PostHeader/index.jsx (hunk 1 above)
Comment (curioussavage) [inline src/app/components/Post/PostHeader/index.jsx]: Actually shouldn't It be using the `OutboundLink` component?

**Comment by curioussavage** [comment]
:eye: @jamesmaa

**Comment by curioussavage** [comment]
👓 @jamesmaa

**Comment by curioussavage** [comment]
per in person review we are sticking with the `a` tag and I removed the logic to link externally when comments are disabled.

we will now link externally based on `post.promoted && !post.isSelf`

**Comment by curioussavage** [comment]
Unless @jamesmaa has anything I will deploy after our spring planning at ~3pm  (fish pending)

**Comment by curioussavage** [comment]
oops :haircut_man:

**Comment by jamesmaa** [comment]
lgtm. thanks for the fix :fish:

**Comment by nramadas** [comment]
also :fish:

**Pull request closed without merging**